examples/timeline/editing/tooltipOnItemChange.html

```diff
@@ -0,0 +1,81 @@
+<html>
+<head>
+  <title>Timeline | Tooltip on item onUpdateTime Option</title>
+
+  <script src="../../../dist/vis.js"></script>
+  <link href="../../../dist/vis.css" rel="stylesheet" type="text/css" />
+
+  <script src="../../googleAnalytics.js"></script>
+</head>
+
+<body>
+
+<h1>Timeline Tooltip on item onUpdateTime Option</h1>
+
+<h2>With <code>tooltipOnItemUpdateTime: true</code>
+</h2>
+
+<div id="mytimeline"></div>
+
+<script>
+
+  // create groups
```

Comment:
I personally would prefer a much more simple example without groups and stuff. Just something like the basicExample.